stock_manual_transfer/models/stock_manual_transfer.py

```diff
@@ -63,6 +63,8 @@ class StockManualTransfer(models.Model):
     )
 
     def action_validate(self):
+        if self.state != "draft":
+            return
```

Comment:
How is this possible?

Reply:
"I don't have an idea how to reproduce the error. But is an issue reported.
 Please remember that this module is already in use but was installed from the Client Road main app (https://git.vauxoo.com/vauxoo/7llantas)
 
 The PM's group has declared it as an issue that cannot be replicated, but a couple of times, the transfers originated from this module has been found duplicated.

For more context, please refer to the comments by Julio Zerna and Zuly Lira in helpdesk ticket 81066. They are the authors of the return early suggestion as an answer of this issue

https://www.vauxoo.com/web#id=81066&cids=1&menu_id=1431&model=project.task&view_type=form